doc/solver/assets/solver_producer.rst

```diff
@@ -0,0 +1,12 @@
+.. _solver-producer:
+
+Solver producer class
```

Comment:
The name deviates from the names used in code  (Solver.ProductionAsset).  As this is dev-documentation, it's important to keep this in sync!